index.html

```diff
@@ -622,11 +622,13 @@ <h1>Streaming GCN<br>Notices in Python</h1>
 
     <section>
       <div class="image-and-caption">
-        <img src="img/screen-shot-07-step-4.png" width="900" alt="Screen shot">
+        <img src="img/screen-shot-07-step-4.png" width="850" alt="Screen shot">
```

Comment:
```suggestion
        <img src="img/screen-shot-07-step-4.png" width="900" alt="Screen shot">
```

I'll supply new screen shots that match the dimensions of the original ones.